test/parser/statements/static-block.ts

```diff
@@ -0,0 +1,143 @@
+import { Context } from '../../../src/common';
+import { pass, fail } from '../../test-utils';
+
+describe('Next - Class static initialization block', () => {
```

Comment:
good to have a few more tests. afaict, the test262 has many failing cases. https://github.com/tc39/test262/commit/afe217b318df5f197e64b30a8b5a4b391c777359

we can try add the test262 at some point, but surely it should be another PR, and won't block merging this one.